docs/assets/themes/zeppelin/js/toc.js

```diff
@@ -0,0 +1,98 @@
+// https://github.com/ghiculescu/jekyll-table-of-contents
```

Comment:
Could you add license entry of toc.js into `LICENSE` file and make a copy of license under `licenses` directory, while this file will be included in our source distribution?